js/apps/system/_admin/aardvark/APP/frontend/js/collections/arangoCollections.js

```diff
@@ -167,7 +167,11 @@
         data.shardKeys = object.shardBy;
       }
       if (object.replicationFactor) {
-        data.replicationFactor = JSON.parse(object.replicationFactor);
+        if (typeof object.replicationFactor === 'number' && frontendConfig.isEnterprise) {
+          data.replicationFactor = JSON.parse(object.replicationFactor);
+        } else {
+          data.replicationFactor = object.replicationFactor;
+        }
       }
```

Comment:
> Jep, this seems to be wrong. 

Option 1: Should address if the value is a numeric value and then parse it: "2" to 2 (data object).
Option 2: Should just set e.g. "satellite" to "satellite" in (data object).